packages/cli/src/environments/sourceControl/sourceControlGit.service.ee.ts

```diff
@@ -191,8 +191,8 @@ export class SourceControlGitService {
 		if (!this.git) {
 			throw new Error('Git is not initialized (setGitUserDetails)');
 		}
-		await this.git.addConfig('user.email', name);
-		await this.git.addConfig('user.name', email);
+		await this.git.addConfig('user.email', email);
```

Comment:
wasn't this already fixed [here](https://github.com/n8n-io/n8n/pull/6839)?

Reply:
yup